druid-shell/src/backend/windows/window.rs

```diff
@@ -666,12 +722,99 @@ impl MyWndProc {
                         }
                     }
                 },
+                DeferredOp::SetRegion(region) => {
+                    unsafe {
+                        match region {
+                            Some(region) => {
+                                let (x_offset, y_offset, client_width) =
+                                    self.get_client_area_specs(hwnd);
+                                let win32_region: HRGN = CreateRectRgn(0, 0, 0, 0);
+
+                                // Add header if there is a frame
+                                if self.has_titlebar() {
+                                    let region_tmp = win32_region;
+                                    let header_rect = CreateRectRgn(0, 0, client_width, y_offset);
+                                    CombineRgn(win32_region, header_rect, region_tmp, RGN_OR);
+                                }
+
+                                let scale = self.scale();
+                                for rect in region.rects() {
+                                    // Create the region as win32 expects it. Round down for low coords and up for high coords
+                                    let region_part = CreateRectRgn(
```

Comment:
`CreateRectRgn` can fail.

Also the resulting `region_part` needs to be fed to `DeleteObject` after we're done combining it.